Tests/RequestTests.swift

```diff
@@ -0,0 +1,211 @@
+//
+//  RequestTests.swift
+//  HyperspaceTests
+//
+//  Created by Tyler Milner on 6/26/17.
+//  Copyright © 2017 Bottle Rocket Studios. All rights reserved.
+//
+
+import XCTest
+import Hyperspace
+
+class RequestTests: XCTestCase {
+    
+    // MARK: - Tests
+    
+    func test_SimpleGETRequestWithoutQueryParametersOrHeaders_GeneratesCorrectURLRequest() {
+        let request: Request<String, AnyError> = .simpleGET
+        assertParameters(for: request)
+    }
+    
+    func test_SimpleGETRequestWithHeaders_GeneratesCorrectURLRequest() {
+        var request: Request<String, AnyError> = .simpleGET
+        request.headers = [.contentType: .applicationJSON]
+        
+        assertParameters(headers: ["Content-Type": "application/json"], for: request)
+    }
+    
+    func test_SimplePOSTRequestWithData_GeneratesCorrectURLRequest() {
+        let bodyData = "Test".data(using: .utf8)!
+
+        var request: Request<String, AnyError> = .simplePOST
+        request.body = HTTP.Body(bodyData)
+        
+        assertParameters(method: "POST", body: bodyData, for: request)
+    }
+    
+    func test_Request_EmptyResponseInit() {
+        _ = EmptyResponse()
+        XCTAssert(true)
+    }
+    
+    func test_RequestWithoutExplicitCachePolicyAndTimeout_ReturnsDefaultCachePolicyAndTimeout() {
+        let request: Request<EmptyResponse, AnyError> = .cachePolicyAndTimeoutRequest
+        XCTAssert(request.cachePolicy == .useProtocolCachePolicy)
+        XCTAssert(request.timeout == 30)
```

Comment:
Also know you didn't write this one, but let's go ahead and improve it so that it's using non-default defaults.
```suggestion
        let timeout = 1
        let cachePolicy: URLRequestCachePolicy = . returnCacheDataDontLoad
        Request.defaultTimeout = timeout
        Request.defaultCachePolicy = cachePolicy
        
        let request: Request<EmptyResponse, AnyError> = .cachePolicyAndTimeoutRequest
        XCTAssert(request.cachePolicy == cachePolicy)
        XCTAssert(request.timeout == timeout)
```
That's just freehanded so you might need to correct a few things so that it will compile.

Reply:
Also, the new multi-line comment feature is 🔥 !

Reply:
Fixed